### PR TITLE
socket rooms added

### PR DESCRIPTION
**Pull request by devongillis**

### CODE REVIEW

File: poker-pals/src/components/pokerComponents/RaiseToggle.vue [hunk 1]
```diff
@@ -3,8 +3,8 @@
         <div class="raiseButtonOuter" id="buttonMinus" v-on:click="decrementRaise()">
             <div class="raiseButton centerText">-</div>
         </div>
-        <div class="raiseValue" id="raiseValue"> {{ bigBlind }}</div>
-        <input class="raiseScroll" id="slider" type="range" v-bind:min="bigBlind" v-bind:max="max" v-bind:value="bigBlind"> <!--TODO: set these using props-->
+        <div class="raiseValue">{{ raiseValue }}</div>
+        <input class="raiseScroll" id="slider" type="range" v-bind:min="bigBlind" v-bind:max="bigBlind*20" v-bind:value="raiseValue"> <!--TODO: set these using props-->
```
Comment (wpasadilla) [inline poker-pals/src/components/pokerComponents/RaiseToggle.vue]: Not sure if this another one of those magic number cases 

bigBlind*20 

Note: it was not brought up during the last pull request.

Comment (devongillis) [reply poker-pals/src/components/pokerComponents/RaiseToggle.vue]: nice catch

File: poker-pals/src/components/Poker.vue [hunk 2]
```diff
@@ -217,8 +208,8 @@ export default {
             });
             // TODO: later use v-bind and not getElementById
             this.checkFold = false;
-            let b = document.getElementById("buttonCheckFold");
-            b.style.backgroundColor = '#ffffff'; // color theme for button not selected
+            //let b = document.getElementById("buttonCheckFold");
+            //b.style.backgroundColor = '#ffffff'; // color theme for button not selected
```
Comment (darbrius) [inline poker-pals/src/components/Poker.vue]: Don't forget to remove commented code

File: server/app.js [hunk 3]
```diff
@@ -62,13 +62,19 @@ let profiles = [
     {userID: 1000, name: 'Mark123', chips: 20000, icon: 'player_icon_1'},
     {userID: 2000, name: 'John456', chips: 40000, icon: 'player_icon_1'},
     {userID: 3000, name: 'Luke854', chips: 60000, icon: 'player_icon_1'},
-    {userID: 4000, name: 'Jane027', chips: 80000, icon: 'player_icon_1'},
-    {userID: 5000, name: 'Stan693', chips: 10000, icon: 'player_icon_1'},
+    {userID: 4000, name: 'Jane027', chips: 180000, icon: 'player_icon_1'},
+    {userID: 5000, name: 'Stan693', chips: 100000, icon: 'player_icon_1'},
     {userID: 6000, name: 'Mack273', chips: 20, icon: 'player_icon_1'},
 ];
 // the entire list of tables, might need a better way to store them?
```
Comment (wpasadilla) [inline server/app.js]: I had it set up like so using a JS object. Rooms are all given a unique ID and then stored into the rooms object using their ID as a map.  

let rooms = {};
for (let x = 1; x <= 10; x++) {
        //Create x number of rooms 
        const room = {
            id: uuidv4(),
            name: "Table " + x,
            buyin: 20000,
            blinds: "1000 / 2000",
            sockets: []
        }
        rooms[room.id] = room;
    }

File: server/app.js [hunk 4]
```diff
@@ -78,7 +84,7 @@ function login(socket, msg){ // user sends a log in request and the server check
         return;
     }
     else{// send the current table state
-        socket.emit('tableState', JSON.stringify(tables[0].getTableState()));
+        //socket.emit('tableState', JSON.stringify(tables[0].getTableState()));
```
Comment (darbrius) [inline server/app.js]: More commented code to remove

File: server/app.js (hunk 3 above)
Comment (wpasadilla) [reply server/app.js]: Maybe this is a more scale-able approach? 

Need opinions.

File: server/app.js (hunk 4 above)
Comment (wpasadilla) [inline server/app.js]: Is this still being implemented? 
Or just commented out for testing purposes?

Comment (devongillis) [reply server/app.js]: no, i realized that the user would not be sent the table state on login, they first would see the list of tables. one thing we really need to get done is linking up all the pages together so i can start using real authentication and real profiles. we should address this in tomorrows meeting

File: server/app.js (hunk 3 above)
Comment (devongillis) [reply server/app.js]: we definitely need to address this tomorrow.

File: poker-pals/src/components/Poker.vue [hunk 5]
```diff
@@ -215,10 +205,7 @@ export default {
                 action: action,
                 raiseToValue: this.raiseToValue, // only used if player is raising
             });
-            // TODO: later use v-bind and not getElementById
             this.checkFold = false;
-            let b = document.getElementById("buttonCheckFold");
-            b.style.backgroundColor = '#ffffff'; // color theme for button not selected
```
Comment (aidan-lee) [inline poker-pals/src/components/Poker.vue:221]: Could b have a more descriptive name?

Comment (devongillis) [reply poker-pals/src/components/Poker.vue:221]: that code was removed

Comment (aidan-lee) [reply poker-pals/src/components/Poker.vue:221]: Weird.  Totally saw it as an addition.  My mistake.

**Review by wpasadilla — APPROVED** [review]

**Pull request merged**